finality-aleph/src/new_network/connection_manager/session.rs

```diff
@@ -197,22 +220,24 @@ mod tests {
 
     const NUM_NODES: usize = 7;
 
-    async fn keyboxes() -> Vec<KeyBox> {
-        let num_keyboxes = NUM_NODES;
+    async fn keyboxes_components() -> Vec<(Option<(NodeIndex, AuthorityPen)>, AuthorityVerifier)> {
```

Comment:
I kinda hate this name, since we don't really have any keyboxes here anymore. `crypto_basics`? I dunno, maybe you have some better ideas?

Reply:
Also doesn't make sense to return always `Some` with an `Option` – I would suggest packing it into `Some` outside of this function if needed.

Also-also, this only needs to return a single `AuthorityVerifier` – they are all identical. If it doesn't implement `Clone` then it should.

Reply:
Done